pkg/util/cloudproviders/cloudfoundry/cccache.go

```diff
@@ -463,7 +463,7 @@ func (ccc *CCCache) listApplications(wg *sync.WaitGroup, appsMap *map[string]*cf
 				sidecars, err := ccc.ccAPIClient.ListSidecarsByApp(query, app.GUID)
 				if err != nil {
 					log.Errorf("Failed listing sidecars from cloud controller: %v", err)
-					return
+					continue
```

Comment:
Why do we `continue` here but not in other places where we call the API?

Reply:
Since sidecar tag collection is an extra feature, we don't want it to break our basic tagging in case something goes wrong with the requests.